documentation/assemblies/assembly-deploying-auth-services.adoc

```diff
@@ -0,0 +1,22 @@
+// Module included in the following assemblies:
+//
+// assembly-installing-manual-steps.adoc
+
+[id='deploying-auth-services-{context}']
+= Deploying authentication services
+
+{ProductName} requires at least one authentication service to be deployed. 
+ifndef::AuthServices[]
```

Comment:
Maybe call this attribute NoneAuthServices.

Reply:
@lulf good idea! Done.